sdk/master_board_sdk/README.md

```diff
@@ -42,3 +42,29 @@ to run the example run:
 sudo ./bin/exec MY_INTERFACE
 ```
 where MY_INTERFACE is the name of the network interface used to connect to the master board.
+
+How to run the Python example
+--------
+
+* Clone the repository: `git clone https://github.com/open-dynamic-robot-initiative/master-board.git`
+
+* Get into the repository: `cd master-board`
+
+* Set the number of controlled drivers by opening `sdk/master_board_sdk/include/master_board_sdk/defines.h` and tuning the `N_SLAVES_CONTROLED` constant: `N_SLAVES_CONTROLED 4` if you are using 4 driver boards (1 per leg)
+
+* In `sdk/master_board_sdk/example/example.py`, at the start of the script, set the number of controled drivers to the value you put in `defines.h`: `N_SLAVES_CONTROLED = 4  # Current number of controled drivers`
```

Comment:
If one has to change something in a file managed by git for a particular setup, there are chances to commit that by mistake.
This also make it impossible to distribute a binary package of this project that would work for every case.

Reply:
But it's a separate issue.

Reply:
Agreed, This value should only be on the example files.